twingate/resource_connector.go

```diff
@@ -129,15 +115,22 @@ func resourceConnectorRead(ctx context.Context, resourceData *schema.ResourceDat
 		return diag.FromErr(err)
 	}
 
-	if err := resourceData.Set("name", connector.Name); err != nil {
+	if err := resourceData.Set("name", string(connector.Name)); err != nil {
 		return diag.FromErr(fmt.Errorf("error setting name: %w ", err))
 	}
 
+	var connectorRemoteNetworkID string
 	if connector.RemoteNetwork != nil {
```

Comment:
Is this check necessary? 
`remoteNetwork` is a required field on Connector. There is no way you reached here and it's `nil` because you wouldnt have gotten an `err` if server returned such response
<img width="389" alt="Screen Shot 2022-10-14 at 5 15 53 PM" src="https://user-images.githubusercontent.com/205185/195960103-51444322-50a6-4c2e-9782-004e77a676ec.png">

Reply:
I see your point, yes we can simplify this code

Reply:
fixed